opsdroid/core.py

```diff
@@ -286,12 +286,12 @@ def setup_skills(self, skills):
 
         """
         if "parsers" in self.config:
-            parsers = self.config["parsers"] or []
-            rasanlu = [p for p in parsers if p["name"] == "rasanlu"]
-            if len(rasanlu) == 1 and (
-                "enabled" not in rasanlu[0] or rasanlu[0]["enabled"] is not False
+            parsers = self.config["parsers"] or {}
```

Comment:
```suggestion
            parsers = self.config.get("parsers", {})
```

Reply:
So interesting thing here. If we do this change then the next line throw out an `AttributeError: 'NoneType' object has no attribute 'get'` when we do `self.config.get("parsers", {})` this expression is evaluating as none (i'm assuming because of the empty dict) whilst the or makes a difference (for some reason).

Leaving the line as it is works just fine 🤷‍♂️

Reply:
Weird! No worries.